tensorflow/buildspec.yml

```diff
@@ -1,7 +1,7 @@
   account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
   region: &REGION <set-$REGION-in-environment>
   framework: &FRAMEWORK tensorflow
-  version: &VERSION 2.1.1
+  version: &VERSION 2.2.1
```

Comment:
Reviewer note: The changes on this file will need to be reverted when PR review is complete.